docs/site/decorators/Decorators_openapi.md

```diff
@@ -427,6 +427,52 @@ export class SomeController {
 }
 ```
 
+#### anyOf, allOf, oneOf, not
```

Comment:
It would be nice to see another PR for this documentation. At lease a separate commit.

Reply:
This should have been in my `complex-x-ts-types` PR, I'll separate it out :)